IFC4x3/Properties/i/IsArcSuppressed_2bj3cZYAD7qvpndHNlg9vP/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="IsArcSuppressed_2bj3cZYAD7qvpndHNlg9vP" Name="IsArcSuppressed" UniqueId="a5b439a3-88a3-47d3-9cf1-9d15efa89e59" PrimaryDataType="IfcBoolean" />
```

Comment:
description is: ability to supress arc. e.g. this device is able to suppress arcs.
name suggests an action placed upon the device, e.g. this device is arc supressed by something.

if description is correct consider: `IsArcSuppressing` or maybe `CanArcSuppress`

or update the description to:
> Indicates whether the element has arc suppression